src/components/SettingsMenu.js

```diff
@@ -0,0 +1,283 @@
+import React, { Component } from "react";
```

Comment:
This whole file is actually factored out from GridLayout.css.
Only the parts below commented that are concerning training are new.